### ISSUE TITLE
WCAG Failure: The html language identifier is missing in the language option drop down menu

### ISSUE THREAD

**Issue by patdunlavey** [ProjectMirador/mirador#3919]
This issue description is from an external accessibility audit of our Mirador 3 install
- Failed WCAG Success Criteria: 3.1.2 Language of Parts (AA)
- Description: The html language identifier is missing for the various languages in the language option drop down menu.
- Recommendation: Add a language identifier to each language option.
